Formula/pueue.rb

```diff
@@ -0,0 +1,67 @@
+class Pueue < Formula
+  desc "Command-line tool for managing long-running shell commands"
+  homepage "https://github.com/Nukesor/pueue"
+  url "https://github.com/Nukesor/pueue/archive/v0.1.4.tar.gz"
+  sha256 "8e156d1037a88448219d9c5bdbc844799e616604985c86a83541f0a23481f302"
+  head "https://github.com/Nukesor/pueue.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    rm bin/"completions_pueue"
```

Comment:
Just released v1.0.5 and removed this binary.
This has now been added to the normal build process with `build.rs` and will be executed when calling `cargo build`